resources/content/articles/physik_grav.html

```diff
@@ -0,0 +1,500 @@
+    Torsionsbandes vorgesehene Drehknopf beschädigt. Für sinnvolle
+    physikalische Messungen war das Gerät nicht mehr brauchbar. Am Ende
+    des Schuljahrs 2004/2005 begeisterten sich einige Schüler der
+    damaligen Klasse 11b für das von mir angedachte Projekt, unsere
+    empfindliche Drehwaage zu reparieren und Messungen von G außerhalb des
+    Unterrichts durchzuführen. Es dauerte geraume Zeit, bis ein neuer,
+    geeigneter Metallfaden aus Bronze bestellt war und geliefert werden
+    konnte. Für den Einbau des filigranen Bronzefadens in die Drehwaage
+    konnten wir den Optikermeister Herrn Fritz Nidermayer aus Neumarkt
+    gewinnen. Er bot uns bei allen Pannen und Problemen seine großzügige
+    Unterstützung an. Bei dieser Gelegenheit möchte ich ihm auch im Namen
+    der Schülerinnen und Schüler nochmals herzlich für seine
+    Hilfsbereitschaft danken. Im neuen Schuljahr 2005/2006 bildete sich
+    spontan eine aus zwei Schülern der ehemaligen 11b bestehende
+    Arbeitsgruppe (Alexander Proksch und Sebastian Obermeyer). Die beiden
+    Kollegiaten wollten nach der Reparatur der Drehwaage genaue Messungen
+    der Gravitationskonstanten durchführen. Nachdem noch eine massive
+    Steinplatte als Standort für Gravitationsdrehwaage an der Wand des
+    Physiksaals befestigt worden war, konnten wir im Januar 2006 eine
+    erste erfolgreiche Messung durchführen. Dann löste sich der dünne
+    Metallfaden der Drehwaage erneut aus der Halterung. Nach dem
+    Wiedereinbau hatten wir unerklärlicherweise große Schwierigkeiten, das
+    Präzisionsinstrument mit der erforderlichen Genauigkeit zu
+    justieren. Zeitaufwändige Justierversuche und mehr oder weniger
+    erfolglose Messungen zogen sich bis in den Januar 2007 hin. In den
+    Weihnachtsferien gelang es dann endlich, die Drehwaage so
+    einzustellen, dass sie ab Januar 2007 zufriedenstellend für längere
+    Zeit funktionierte – bis zum heutigen Tag (26. Juli 2007).
+</p>
+
+<p>
+    Alexander und Sebastian führten bis unmittelbar vor Beginn der
+    intensiven Vorbereitungsphase für das Abitur einige Messungen
+    durch. Sie bestimmten jeweils nach einer sorgfältigen Beobachtung der
+    Gleichgewichtslage des Pendels möglichst genau seine Schwingungsdauer
+    T, die Länge L des Lichtzeigers zur Beobachtung der Auslenkung des
+    Pendels und den Abstand zwischen den beiden Gleichgewichtslagen des
+    Systems (Endausschlag S) auf einem Beobachtungsschirm.
+</p>
+
+<p>
+    Inzwischen hatte ich wieder eine 11. Klasse (11e im Schuljahr
+    2006/2007) in Physik. Nun konnten wir die Messung der
+    Gravitationskonstanten ohne große Vorbereitungszeit als ergänzendes
+    Unterrichtsprojekt durchführen. Die Schüler Daniel Gottschalk, David
+    Hink, Dominik Kraus, Michael Mosburger, Christoph Summersamer und
+    Amadeus Trubig (siehe Foto 1) nahmen die Messwerte auf.
+</p>
+
+<p>
+    <v-image name="grav_schueler1" alt="Foto 1" />
```

Comment:
v-image sollte eigentlich nicht in paragraphen sein, es sei denn dies hat einen bestimmten Grund?